### PR TITLE
Expose new SKUs for VirtualNetworkGateway ER and VPN VMSS

### PR DESCRIPTION
**Pull request by ritwikbasu**
Expose new SKUs for VirtualNetworkGateway ER and VPN VMSS

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by ritwikbasu** [comment]
The REST-API PR is https://github.com/Azure/azure-rest-api-specs/pull/3069

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.0.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: This does not look like a breaking change. You can simply set it as `19.0.1-preview` instead

Comment (dsgouda) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Also, update the `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please generate the code using generate.ps1 script and commit changes to the .txt log file

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj (hunk 1 above)
Comment (ritwikbasu) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Fixed in second commit.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge on CIs passing

File: src/SDKs/_metadata/network_resource-manager.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
-2018-04-26 20:59:33 UTC
+2018-05-15 23:11:28 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
-Branch:      master
-Commit:      b9721ae4c9f04914cc89260b8339dc33372f835b
+Branch:      Network-2018-05-01
```
Comment (dsgouda) [inline src/SDKs/_metadata/network_resource-manager.txt]: Apologize for missing this, we cannot accept PRs for sdks generated from a branch other than master. If your changes have already been merged in the Network May REST spec, you don't need this PR
Please talk to @MikhailTryakhov if you have any doubts

**Comment by dsgouda** [comment]
These changes will be introduced with the Network May REST spec sdk

**Pull request closed without merging**